de/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: news_post
+title: "Ruby 2.3.0-preview1 veröffentlicht"
+author: "naruse"
+translator: "Marvin Gülker"
+date: 2015-11-11 14:00:00 +0000
+lang: de
+---
+
+Wir freuen uns, die Veröffentlichung von Ruby 2.3.0-preview1
+bekanntgeben zu können.
+
+Ruby 2.3.0-preview1 ist die erste Vorschau auf Ruby 2.3.0, in die
```

Comment:
Das wäre zwar näher am Original, aber m.E. ist die idiomatisch schönere Version als einzelner, fließender Satz einer bloßen 1:1-Übersetzung überlegen, weil sie die Leserlichkeit fördert und den Text als verwobendes Gebilde und nicht als Satzstakkato erscheinen lässt (das sich in den englischen Texten leider zu oft findet, auch, weil die Sprache es nicht immer anders hergibt). Gibt es zwingende Gründe, die gegen derartige minimale, auf die Zielsprache zugeschnittenen Änderungen sprechen?